### ISSUE TITLE
Tracking citations from datasets

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#2405]
**Tracking citations from datasets**

For the Tracking Invasive Alien Species (TrIAS) projects, we are creating "occurrence cubes" derived from GBIF downloads, see 
`https://doi.org/10.5281/zenodo.3635510.`

The source GBIF downloads are referenced in "related identifiers" as with a "derived from" relation. We now have 2 versions of the cube, both with different source downloads.

I was wondering if this type of citation could be taken into account in GBIF Literature tracking: `https://www.gbif.org/literature-tracking?`

-----

Github user: @peterdesmet
User: [See in registry](https://www.gbif.org/api/feedback/user/bc004c70dfeacd0166eb5251fe21824e:84eb5bf9d13d8d57596840d6eeea1ef4349e7897241815bd8df7444923e0962be76d648f964a5c1a17c02ae3e6e2bff237615f0594ad870080d617aff21808cc)
System: Chrome 79.0.3945 / Mac OS X 10.14.6
Referer: https://www.gbif.org/user/profile
Window size: width 1266 - height 717
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-02-04T14:30:20.585Z',mode:absolute,to:'2020-02-04T14:36:20.585Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-02-04T14:30:20.585Z',mode:absolute,to:'2020-02-04T14:36:20.585Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by dnoesgaard** [comment]
Hey @peterdesmet, thanks for raising this. With the GBIF download DOIs in relatedIdentifiers we should be able to get these "automatically" through Event Data, so kudos for adding those links. Unfortunately, Event Data isn't super stable or reliable at the moment, but I'll make sure these get added right away.

**Comment by dnoesgaard** [comment]
https://www.gbif.org/resource/search?q=occurrence%20cubes&contentType=literature&relevance=GBIF_USED

**Comment by peterdesmet** [comment]
So the `relatedIdentifiers` harvesting was implemented already?! In any case, thanks for adding. Hope to add more in the future and see them harvested automatically.

**Comment by peterdesmet** [comment]
Note, we also have datasets that reference GBIF datasets as `source of`. See https://doi.org/10.5281/zenodo.3541811 Are these harvested or only certain relations?

**Comment by dnoesgaard** [comment]
Yes and no. We do consume Event data that is based on relatedIdentifiers, however, it's very sparse and until now I've only used links from Crossref, i.e. papers. I'll have to figure out a way of including dataset-dataset links—probably by looking at specific relationType...

(I'll reopen this one as a reminder)
